frontend/src/app/highlight/highlight.component.ts

```diff
@@ -1,4 +1,6 @@
 import {Component, Input, OnInit} from '@angular/core';
```

Comment:
Can you comment how this patch avoids memory leaks?

Reply:
is also mentioned here (https://dev.to/theoklitosbam7/avoid-memory-leaks-in-angular-5gla) in the async template directive